### ISSUE TITLE
New customer emails still sending when disabled

### ISSUE THREAD

**Issue by bsstaveley** [dotmailer/dotmailer-magento2-extension#85]

**Comment by cdiacon** [comment]
related ?!? https://github.com/dotmailer/dotmailer-magento2-extension/issues/43

**Comment by cdiacon** [comment]
@adeelq 
So this can be replicated in 2.0.6 and 2.0.7 : 

di.xml : 

change this : 

```
   <preference for="Magento\Customer\Model\Customer" type="Dotdigitalgroup\Email\Model\Customer"/>
```

to this : 
```
<preference for="Magento\Customer\Model\AccountManagement" type="Dotdigitalgroup\Email\Model\Customer"/>

```


And 

Dotdigitalgroup\Email\Model\Customer extends \Magento\Customer\Model\Customer


to : 

Dotdigitalgroup\Email\Model\Customer extends \Magento\Customer\Model\AccountManagement

**Comment by adeelq** [comment]
Thanks for this. @cdiacon did you tested above. I do not have older version any more. Need to install older version M2.

**Comment by cdiacon** [comment]
Last fix was done here https://github.com/dotmailer/dotmailer-magento2-extension/issues/43

but it's not working for 2.0.6 and 2.0.7, this may be a solution to fix for versions  < 2.1.0

**Comment by bsstaveley** [comment]
Issue still exists on Magento 2.1 but will be fixed in a future connector release also version 2.1
